platforms/aarch64-rk3588j.toml

```diff
@@ -0,0 +1,68 @@
+# Architecture identifier.
+arch = "aarch64"
+# Platform identifier.
+platform = "aarch64-rk3588j"
+# Platform family.
+family = "aarch64-rk3588j"
+
+# Base address of the whole physical memory.
+phys-memory-base = "0x20_0000"
+# Size of the whole physical memory.
+phys-memory-size = "0x800_0000"     # 128M
+# Base physical address of the kernel image.
+kernel-base-paddr = "0x48_0000"
+# Base virtual address of the kernel image.
+kernel-base-vaddr = "0x0000_0000_0048_0000"
+# Linear mapping offset, for quick conversions between physical and virtual
+# addresses.
+phys-virt-offset = "0x0000_0000_0000_0000"
+# Kernel address space base.
+kernel-aspace-base = "0x0000_0000_0000_0000"
```

Comment:
without virtualization, we'd better put kernel at high address
see `aarch64-bsta1000b.toml` and `aarch64-raspi4.toml`

Reply:
Ok.